packages/runtime/src/context.ts

```diff
@@ -22,6 +22,8 @@ export interface RuntimeContext<Environment extends RuntimeEnvironment = Integra
      * Authenticated client to the GitBook API.
      */
     api: GitBookAPI;
+
+    waitUntil: FetchEvent['waitUntil'];
```

Comment:
I'm not sure if overall this is a good structure on how we should be exposing parts of the event's request 🤔

Reply:
Why not?